runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -922,12 +922,13 @@ public void translate(StreamingPubsubIORead<T> transform, TranslationContext con
               ((NestedValueProvider) overriddenTransform.getSubscriptionProvider()).propertyName());
         }
       }
-      if (overriddenTransform.getTimestampLabel() != null) {
+      if (overriddenTransform.getTimestampAttribute() != null) {
```

Comment:
Yup, I'll coordinate with @melap on what to do about all these style-guide-related API changes next week.